docs/SORMAS2SORMAS.md

```diff
@@ -0,0 +1,136 @@
+
+#### Server Descriptors
+
+A server descriptor is a struct which contains the information needed to connect to a SORMAS instance (see the
+`SormasServerDescriptor` class). It consists of the S2S id, a human-readable name used in the UI, and the hostname.
+Descriptors centrally distributed via a dedicated `etcd` keyspace (e.g., `/s2s/`).
+
+```json
+{
+  "id":"2.sormas.id.sormas_a",
+  "name":"sormas_a_org_name",
+  "hostName":"sormas_a:6080"
+}
+```
+
+#### Keystore
+
+Each instance receives a dedicated x509 certificate and private key pair which is used to encrypt all S2S communication.
+The certificate is signed by a common S2S CA. The certificate and private key are stored in a `pkcs12` keystore under the
+S2S id of the instance. See [here](https://github.com/hzi-braunschweig/S2S-Testbed/blob/main/docker/certs/sormas_a/generate-cert-a.sh)
+for an example of how to generate a keystore.
+
+#### Truststore
+
+Each instance receives a truststore which contains the S2S CA certificate under the `sormas2sormas.rootCaAlias`.
+See [here](https://github.com/hzi-braunschweig/S2S-Testbed/blob/main/docker/certs/ca/generate_ca.sh) for an example of
+how to generate a truststore.
+
+#### Properties
+
+The following properties are required to be set in the `sormas.properties` file:
+
+| Property                                         | Description                                                                                                                                          |
+|--------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `central.oidc.url`                               | URL of the OIDC server authenticating instances and authorizing S2S requests.                                                                        |
+| `central.etcd.host=`                             | The hostname of the etcd instance providing data like server descriptors.                                                                            |
+| `central.etcd.clientName`                        | The client name of the instance used in authentication towards etcd.                                                                                 |
+| `central.etcd.clientPassword`                    | The client password of the instance used in authentication towards etcd.                                                                             |
+| `central.etcd.caPath`                            | The full filesystem path to the CA trusted by etcd clients when creating a TLS connection to the etcd server.                                        |
+| `central.location.sync`                          | If set to true, all infrastructure data from the central server will be synchronized into the local SORMAS database at startup and on a daily basis. |
```

Comment:
@jp-sgent Currently not, but it would be easy to add a button to the UI or to add a ReST endpoint. If needed, please create an issue giving some details on the use case.